tools/doc/html.js

```diff
@@ -30,12 +30,12 @@ var gtocPath = path.resolve(path.join(
 var gtocLoading = null;
 var gtocData = null;
 
-function toHTML(input, filename, template, nodeVersion, cb) {
-  if (typeof nodeVersion === 'function') {
-    cb = nodeVersion;
-    nodeVersion = null;
-  }
-  nodeVersion = nodeVersion || process.version;
+/**
+ * opts: input, filename, template, nodeVersion.
+ */
+function toHTML(opts, cb) {
+  var template = opts.template;
+  var nodeVersion = opts.nodeVersion || process.version;
```

Comment:
Thanks for the feedback @addaleax. Yes, they could be. FWIW when I first PRed those changes there was no `const` in that file and after the changes were first merged in #3888 and reverted in #6680 I didn't want to rock the boat with anything other than changes that had already been previously +1ed (since the original PR had a hard enough time landing). Also FWIW there's a bunch of other `var` in the file that could be `const`. The specific vars you commented on are each only referenced once, so honestly probably the best thing is to just eliminate those bindings and use `opts` where those are referenced. The same probably goes for at least some of the `var`s in `render()`.

Let me know if you want me to make a change either way.